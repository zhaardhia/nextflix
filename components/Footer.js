import styles from '../styles/Footer.module.css'

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <p>Copyright &copy; Campflix 2022</p>
    </footer>
  )
}
  
export default Footer