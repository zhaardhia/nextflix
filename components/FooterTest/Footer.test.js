import { render, screen } from "@testing-library/react";
import Footer from "../Footer";
import '@testing-library/jest-dom'


describe("Footer component", () => {
  it("renders successfully", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});