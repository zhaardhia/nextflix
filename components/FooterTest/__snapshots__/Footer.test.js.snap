// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders successfully 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      Copyright © Campflix 2022
    </p>
  </footer>
</div>
`;

exports[`Header component renders successfully 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      Copyright © Campflix 2022
    </p>
  </footer>
</div>
`;
